ci-operator/step-registry/cucushift/upgrade/check/capability/cucushift-upgrade-check-capability-commands.sh

```diff
@@ -329,9 +336,28 @@ if [[ "${baselinecaps_from_cluster}" == "v4."* ]]; then
     else
         echo "migrated_caps is not empty."
         for key in "${!migrated_caps[@]}"; do
-            if [[ $source_enabled_caps == *$key* ]] && [[ $target_enabled_caps != *${migrated_caps[$key]}* ]]; then
-                expected_enabled_caps="${expected_enabled_caps} ${migrated_caps[$key]}"
-                expected_implicit_caps="${expected_implicit_caps} ${migrated_caps[$key]}"
+            # Handle the scenario of multiple caps in key
+            IFS='+' read -r -a key_caps <<< "$key"
```

Comment:
Why here we need to set `IFS`? If I understand right, the `key` does not have `+` char, but its value has the char, right?

Reply:
Ignore it, I got the idea.